src/main.js

```diff
@@ -1,31 +1,111 @@
+  extraChild.classList.add('modal-age-select');
+  extraChild.innerHTML = `
+  <option value="1" selected>0 years old</option>
+  <option value="2">1 year old</option>
+  <option value="3">2 years old</option>
+  <option value="4">3 years old</option>
+  <option value="5">4 years old</option>
+  <option value="6">5 years old</option>
+  <option value="7">6 years old</option>
+  <option value="8">7 years old</option>
+  <option value="9">8 years old</option>
+  <option value="10">9 years old</option>
+  <option value="11">10 years old</option>
+  <option value="12">11 years old</option>
+  <option value="13">12 years old</option>
+  <option value="14">13 years old</option>
+  <option value="15">14 years old</option>
+  <option value="16">15 years old</option>
+  <option value="17">16 years old</option>
+  <option value="18">17 years old</option>
+  `;
+  addChildInfo.append(extraChild);
+}
+
+function openModal() {
+  modalConditions.style.display = 'block';
+}
+function closeModal() {
+  modalConditions.style.display = 'none';
+}
+inputConditions.addEventListener('click', openModal);
+
+window.addEventListener('click', (event) => {
+  if (!event.target.contains('.main-form-modal')) {
+    closeModal;
+  }
+});
 
-const createElement =(tagName, className)=>{
-    const elem = document.createElement(tagName);
-    elem.classList.add(className);
-    return elem;
+function counterPlus(item, total) {
+  let count = Number(item.innerHTML);
+  if (count < total && count >= 0) {
+    count++;
+    item.innerHTML = count;
+  }
 }
 
-const render = (data)=>{
-    data.forEach((item)=>{
-        const {name, city, country, imageUrl} = item;
-        const homesCard = createElement('div', 'homes-card');
-        const img = createElement('img', 'homes-image');
-        img.src = imageUrl;
-        img.alt = `${name}`;
-        img.width = '295';
-        img.height ='295';
-
-        const title = createElement('h3', 'homes-image-name');
-        title.textContent = `${name}`;
-
-        const location = createElement('p', 'homes-image-location');
-        location.textContent = `${city}, ${country}`;
-
-        homesCard.append(img, title, location);
-        homesCards.append(homesCard);
-    })
+function counterMinus(item, total) {
+  let count = Number(item.innerHTML);
+  if (count <= total && count > 0) {
+    count--;
+    item.innerHTML = count;
+  }
 }
-render(dataHomes);
+
+upBtnAdults.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterPlus(countBtnAdults, 30);
+});
+
+downBtnAdults.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterMinus(countBtnAdults, 30);
+});
+
+upBtnRooms.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterPlus(countBtnRooms, 30);
+});
+
+downBtnRooms.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterMinus(countBtnRooms, 30);
+});
+upBtnChilds.addEventListener('click', (e) => {
+  e.preventDefault();
+  counterPlus(countBtnChilds, 10);
```

Comment:
You have to have 10 limit for children, hovewer + button still work and add options. You can add disabled for this button for example
<img width="818" alt="Screenshot 2022-07-08 at 12 00 34" src="https://user-images.githubusercontent.com/25488411/177946427-81f87ea4-c58a-4e84-9dc2-d0dc3815afe9.png">

This task is not about css, but try to overlap the second section title

Reply:
Added if statement limiting select further spread after 10 clicks.